psi4/src/psi4/libmints/basisset.h

```diff
@@ -390,7 +396,7 @@ class BasisSet
      * @param py::dict      Python dictionary containing the basis information
      * @param forced_puream Force puream or not
     **/
-    static std::shared_ptr<BasisSet> construct_from_pydict(const std::shared_ptr <Molecule> &mol, py::dict pybs, const int forced_puream);
+    static std::shared_ptr<BasisSet> construct_from_pydict(const std::shared_ptr <Molecule> &mol, py::dict &pybs, const int forced_puream);
```

Comment:
In this case, it'll be undefined in libmints.a, but defined in core.so.  If somebody wanted to link in libmints.a to another application, this could be problematic.  However, the static function is specific to out Python interface, so it's fine for it to only be defined in core.so, IMHO.